### PR TITLE
Fix: show compatible wallets on Multiple Recipients Screen

### PR DESCRIPTION
**Pull request by cmgustavo**
Screen Shot 2022-10-28 at 10 48 57

Screen Shot 2022-10-28 at 10 52 29

### CODE REVIEW

File: src/navigation/wallet/components/SendToAddress.tsx [hunk 1]
```diff
@@ -115,6 +115,7 @@ const SendToAddress = () => {
     keys,
     id,
     currencyAbbreviation,
+    wallet.chain,
```
Comment (gabrielbazan7) [inline src/navigation/wallet/components/SendToAddress.tsx:118]: You can use the extracted `chain` property from the wallet object (line 112)

File: src/navigation/wallet/screens/send/SendTo.tsx [hunk 2]
```diff
@@ -522,6 +522,7 @@ const SendTo = () => {
         keyId,
         address,
         currency: credentials.coin,
+        chain: wallet.chain,
```
Comment (gabrielbazan7) [inline src/navigation/wallet/screens/send/SendTo.tsx:525]: I think it's better to use selectedWallet `chain` value

File: src/navigation/wallet/components/SendToAddress.tsx (hunk 1 above)
Comment (cmgustavo) [reply src/navigation/wallet/components/SendToAddress.tsx:118]: Oh, that's correct. Thanks!

File: src/navigation/wallet/screens/send/SendTo.tsx (hunk 2 above)
Comment (cmgustavo) [reply src/navigation/wallet/screens/send/SendTo.tsx]: Fixed. Thanks!

**Review by gabrielbazan7 — APPROVED** [review]
ACK!

File: src/components/list/WalletRow.tsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ const NestedArrowContainer = styled.View`
 export interface WalletRowProps {
   id: string;
   img: string | ((props: any) => ReactElement);
-  badgeImg?: string | ((props: any) => ReactElement);
+  badgeImg?: string | ((props?: any) => ReactElement);
```
Comment (cmgustavo) [inline src/components/list/WalletRow.tsx:36]: Fixed error:
```
Interface 'KeyWallet' cannot simultaneously extend types 'Wallet' and 'WalletRowProps'.
```

**Pull request merged**